_ide_helper.php

```
@@ -29787,6 +29791,22 @@ public static function end()
+         *
+         * This can be helpful when changing "dependencies" of bootstrappers such as
+         * attributes of the current tenant that are only read once, during bootstrap().
+         *
+         * If tenancy is not initialized, this method is a no-op.
+         *
+         * @static
+         */
+        public static function reinitialize()
+        {
+            /** @var \Stancl\Tenancy\Tenancy $instance */
+            return $instance->reinitialize();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$instance'. (undefined)

(UnusedLocalVariable)